cognite/neat/core/data_classes/config.py

```diff
@@ -51,6 +51,8 @@ class ClientConfig(BaseModel):
     client_name: str = "neat"
     base_url: str = "https://api.cognitedata.com"
     scopes: list[str] = ["project:read", "project:write"]
+    timeout: int = 60
```

Comment:
Suggest renaming this to include unit, sec, min, or hour?
```suggestion
    timeout_sec: int = 60
```

Reply:
Reusing the same naming from CDF SDK . In UI and docs it's mentioned that it's in seconds.